### ISSUE TITLE
[NEWS] Sites that block adblockers seem to be suffering

### ISSUE THREAD

**Issue by SMed79** [reek/anti-adblock-killer#1360]
**Sites that block adblockers seem to be suffering**
https://thestack.com/world/2016/04/21/sites-that-block-adblockers-seem-to-be-suffering/

**Comment by Giwayume** [comment]
Makes sense. An adblock user could like an article and share it on facebook with 100s of their friends and ... oops. That's a lot of potential ad revenue you've missed.
